src/content/en/tools/workbox/get-started/webpack.md

```diff
@@ -313,7 +313,11 @@ that they had an internet connection.
 1. Open `webpack.config.js` again.
 1. Add a `runtimeCaching` property to your Workbox configuration.
    `urlPattern` is a regular expression pattern telling Workbox which
-   URLs to store locally. `handler` defines the caching strategy that Workbox
+   URLs to store locally. *(Please note your regex will be matched
+   against full URLs such as 'https://hacker-news.firebaseio.com' -
+   not against partial URLs. This means you can cache both content from
```

Comment:
The use of the word "not" here is a little problematic to me. I don't think I want to use "full URLs" or "partial URLs" as the main distinction. The idea here is that if your app makes a network request at runtime, and you have a runtime caching handler that matches the regular expression, then that resource gets cached, regardless of its origin.

So how about we rephrase your section to:

```When your app makes a network request at runtime, Workbox caches any request that matches the regular expression in `handler`, regardless of its origin. This means that you can cache content from external sites as well.```

Reply:
Done - though I think it's a loss not being explicit that the URLs are "full" rather than partial.  I burned 2 days thinking workbox was broken, going on the assumption that the regex was being tested against partial URLs, i.e. local URLs.  I reckon others may make the same false assumption; as it's common to only think about local resources when developing a website.

The redraft loses that clarity and I think that may lead others down the same rabbithole.  Essentially I wanted to save them that.